cypress/integration/awards-for-all.js

```diff
@@ -477,11 +477,17 @@ describe('awards for all', function() {
             });
         }
 
-        function fillContact(contact) {
+        function fillContact(contact, type) {
             cy.getByLabelText('First name').type(contact.firstName);
 
             cy.getByLabelText('Last name').type(contact.lastName);
 
+            if (type === 'main') {
```

Comment:
Rather than looking at the `type` it's better to add a check for if the field exists (i.e. these tests are better off working out what to do based on what's on the page, not needing to know about the underlying logic)